themes/fikaworks/layouts/members/list.html.html

```diff
@@ -1,86 +0,0 @@
-{{ define "main" }}
```

Comment:
we could have just done a git mv "stop erasing my history!" :sob:

Reply:
that's ok, reverted and used git mv, I will probably add prettier one in the coming days to keep the indentation consistent.